src/InputGroup/InputGroup.js

```diff
@@ -280,6 +299,7 @@ InputGroup.propDescriptions = {
     addon: 'The value of the add-on.',
     addonPos: 'Location of the add-on relative to the input.',
     inputId: 'Value for the `id` attribute on the `<input>` element.',
+    inputClassName: 'Value for the `classname` attribute on the `<input>` element.',
```

Comment:
Let's copy the current default for classNames - "CSS class(es) to add to the `<input>` element."